src/python/WMCore/WebTools/Page.py

```diff
@@ -378,7 +380,8 @@ def runDas(self, func, data, expires):
 
     keyhash = hashlib.md5()
 
-    keyhash.update(str(results))
+    # TODO: if results is not a bytes/sequence string, this will break!
+    keyhash.update(encodeUnicodeToBytesConditional(results, condition=PY3))
```

Comment:
Why not

```suggestion
    keyhash.update(encodeUnicodeToBytesConditional(str(results), condition=PY3))
```
?

Reply:
Thanks, Dario. I have updated the code as we have discussed over slack.